Cat2Cat/main.m

```diff
@@ -141,7 +144,13 @@ int main(int argc, const char * argv[])
            description:@"Asset catalog(s)"
      blockWithArgument:^(NSString *value) {
          [rawAssetCatalogs addObject:value];
-     }];
+    }];
+    [options addOption:"method-name-prefix"
+                  flag:'m'
+           description:@"Prefix for method names (defaults to ac)"
+     blockWithArgument:^(NSString *value) {
+        methodNamePrefix = [value copy];
+    }];
```

Comment:
I think, given that you're stating the default value in the description, it might as well move up here.  I'm inclined to think the simplification of not having to use the block is also worth the move.